src/config.py

```diff
@@ -39,6 +39,11 @@ def sync_password(self) -> Optional[str]:
         """Returns charm-set sync_password for server-server auth between brokers."""
         return self.charm.get_secret(scope="app", key="sync-password")
 
+    @property
+    def exporter_address(self) -> str:
+        """The port used for JMX exporter."""
+        return "0.0.0.0:9150"
```

Comment:
Maybe it would be best to factor this out where all constants are?

Reply:
I think because this is a property, it's factored out already, here.

Reply:
I kinda like to have all configs close to each other. Literals or config (at the top of the module, after imports) seems both sensible places.

Reply:
I would like to avoid the use of globals across the board. If we need to use them, we can add them to `literals.py`, otherwise it's much more useful to have them pinned within the reference object, i.e `KafkaObject.calculated_var`